extensions/spotify-player/src/nowPlayingMenuBar.tsx

```diff
@@ -12,40 +12,51 @@ import {
   Color,
 } from "@raycast/api";
 import { useEffect, useState } from "react";
-import { currentPlayingTrack, likeCurrentlyPlayingTrack, startPlaySimilar } from "./client/client";
-import { CurrentlyPlayingTrack } from "./client/interfaces";
-import { isSpotifyInstalled, spotifyPlayingState, SpotifyPlayingState } from "./client/utils";
-import { nextTrack, pause, play, previousTrack } from "./controls/spotify-applescript";
+import { likeCurrentlyPlayingTrack, startPlaySimilar } from "./spotify/client";
+import { SpotifyPlayingState, SpotifyState, TrackInfo } from "./spotify/types";
+import { isSpotifyInstalled } from "./utils";
+import { getState, getTrack, nextTrack, pause, play, previousTrack } from "./spotify/applescript";
 
-export default function Main() {
-  const [currentlyPlayingTrack, setCurrentlyPlayingTrack] = useState<CurrentlyPlayingTrack | undefined>();
+export default function NowPlayingMenuBar() {
   const [spotifyInstalled, setSpotifyInstalled] = useState<boolean | undefined>();
-  const [currentSpotifyPlayingState, setCurrentSpotifyPlayingState] = useState<SpotifyPlayingState>(
-    SpotifyPlayingState.Stopped
-  );
+  const [currentlyPlayingTrack, setCurrentlyPlayingTrack] = useState<TrackInfo | undefined>();
```

Comment:
These changes shouldnt change the behaviour but just aim to tidy up the read path.